src/docs/JobPlanner/references/JobPlannerServiceReference.adoc

```diff
@@ -299,4 +299,144 @@ In such case, an *Association failed* notification will be triggered to users su
 manual reactivation for the submissions to restart.
 
-In such case, an *Association failed* notification will be triggered to users subscribed to this event.
\ No newline at end of file
+In such case, an *Association failed* notification will be triggered to users subscribed to this event.
+
+=== Job Planner RBAC management
+
+RBAC, or Role Based Access Control, represents a control model for a system in which each access decision is based on the role to which the user is associated.
+In the Job-Planner, RBAC operations are only performed by Job-Planner admins or associations' owners.
+RBAC operations can be distinguished in two types:
+
+1. Managing an authorization for an association
+2. Managing an authorization for all associations owned by a specific user
+
+Both types of authorizations can be assigned for either a specific user (using her/his username) or a user group (that contains multiple users).
+Only one authorization can be assigned to a username or a user group for each type of RBAC operation.
+However, the user can eventually have multiple authorizations due to a mix of username and user-group related authorizations for an association or a group of them (due to an authorizations resulting from the second type of RBAC operations).
+Furthermore, the created authorizations can be modified or deleted.
+
+An authorization gives the specified user(s) some access permission to the specific association(s).
+We provide four access types that can be authorized:
+
+* Read: The user or user group can only read the association(s) and visualize its planning in the gantt chart.
+* Write: The user or user group have, in addition to the read right, the privilege to modify the association(s)' variables, deactivate or reactivate it(them).
+* Admin: The user or user group have, in addition to the write rights, the ability to delete an association(s).
+Moreover, they can create, modify or delete the authorizations of other users for the association(s).
+* No Access: The user or group of users have no access to the association(s).
+This access type is usually used when the admin wants to authorize a user group except for some specific users, or for example authorize access to her/his all associations except for some of them.
+
+To manage the association authorizations, first you need to select one then a shield-like button above the associations view in the Job-Planner calendar associations portal will be activated, as shown in the figure below.
+
+image::../../images/Association-authorisations-button.png[align=center]
+
+Once you click on it, a dedicated authorization view will open.
+The figure below shows an example.
+
+image::../../images/Association-authorisations-view.png[align=center]
+
+The authorization view consists in two parts:
+
+* First, there are three buttons that allow the admin to refresh the authorizations list, add a new authorization and delete an existing one
```

Comment:
The buttons on top allow to refresh the authorizations list, add a new authorization and delete an existing one